modules/team_planner/spec/features/onboarding/team_planner_onboarding_tour_spec.rb

```diff
@@ -85,7 +85,7 @@
 
   after do
     # Clear session to avoid that the onboarding tour starts
-    page.execute_script("window.sessionStorage.clear();")
+    page.driver.cookies.clear
```

Comment:
`window.sessionStorage` is not the cookie jar, it's a session based local storage. Is that included in the  `cookies.clear` here?

Reply:
I see. So, looks like cuprite (or Ferrum to be precise) doesn't provide access to the `sessionStorage`

```
     Failure/Error: page.driver.execute_script('window.sessionStorage.setItem("key", "value");')

     Ferrum::JavaScriptError:
       DOMException: Failed to read the 'sessionStorage' property from 'Window': Access is denied for this document.
```

However, now that you pointed this out, I noticed that these sessionStorage settings are not changing the spec's behavior at all (in the dev branch). I can remove both the lines that set the sessionStorage item and clear it and it doesn't change the spec's behavior. The way it was currently written doesn't really cause the onboarding tour to start from the backlogs module...